src/reducers/vms.js

```diff
@@ -26,22 +30,27 @@ import {
 } from '_/constants'
 import logger from '../logger'
 import { actionReducer, removeMissingItems } from './utils'
+import { sortFields } from '_/utils'
+import { loadFromLocalStorage } from '_/storage'
+
+const EMPTY_MAP = Immutable.fromJS({})
+const EMPTY_ARRAY = Immutable.fromJS([])
 
 const initialState = Immutable.fromJS({
   vms: {},
   pools: {},
+  filters: Immutable.fromJS(JSON.parse(loadFromLocalStorage('vmFilters')) || {}),
+  sort: Immutable.fromJS({ ...sortFields[0], isAsc: true }),
```

Comment:
> Moving the loads from here to there would be consistent with the rest of the reducers.

I'm not sure that it will be consistent, because we didn't have similar case with using local storage. Also, without moving it to fetched data with no pre-reqs, we can avoid unnecessary dispatching action.